extension/resources/api-extensions.json

```diff
@@ -0,0 +1,18 @@
+[
+  {
+    "key": "ctp-adyen-integration-payment-extension",
+    "destination": {
+      "type": "HTTP",
+      "url": "${ctpAdyenIntegrationBaseUrl}/payments"
+    },
+    "triggers": [
+      {
+        "resourceTypeId": "payment",
+        "actions": [
+          "Create",
+          "Update"
+        ]
+      }
+    ]
+  }
+]
```

Comment:
It looks like we will have only 1 extension. So maybe let's make it an object?

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/4/commits/77f4e9d77bc011cd79fc1f0f0b9d70f4babaeab6